web/impact/impact/tests/test_api_routes.py

```diff
@@ -111,6 +111,41 @@ def test_api_object_get(self):
             response_dict = json.loads(response.content)
             self.assertIn("is_visible", response_dict.keys())
 
+    def test_api_related_object_list_get(self):
+        StartupFactory(is_visible=1, organization__url_slug="test1")
+        StartupFactory(is_visible=1, organization__url_slug="test2")
+        StartupFactory(is_visible=1, organization__url_slug="test3")
+        Permission.objects.get_or_create(
+            content_type=ContentType.objects.get(
+                app_label='mc',
+                model='startup'),
+            codename='view_startup_additional_industries'
+        )
+        view_perm, _ = Permission.objects.get_or_create(
```

Comment:
this looks like a duplication of the previous statement, delete

Reply:
Deleted.